### PR TITLE
[DFFS-156] Build a To-Do App with REACT

### PR DESCRIPTION
**Pull request by CtripleU**

### CODE REVIEW

File: submissions/cumiOyemike/react/to-do-app/src/index.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  /* display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; */
```
Comment (Ireti203) [inline submissions/cumiOyemike/react/to-do-app/src/index.css:25]: Remove commented out code

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/cumiOyemike/react/to-do-app/src/index.css (hunk 1 above)
Comment (CtripleU) [reply submissions/cumiOyemike/react/to-do-app/src/index.css]: Hello, I just made the changes. Kindly review again. Thanks.

File: submissions/cumiOyemike/react/to-do-app/src/components/Todos/Todos.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            id: 1 + Math.random(),
+            completed: this.state.completed,
+            value: this.state.newItem.slice()
+        }
+
+        const searchTerms = [...this.state.searchTerms];
+        searchTerms.push(newItem);
+        this.setState({ searchTerms, newItem: "" });
+
+        this.state.display = 'inline';
```
Comment (ToluAkin) [inline submissions/cumiOyemike/react/to-do-app/src/components/Todos/Todos.js:35]: You are updating state directly. You shouldn't. You can do this instead `this.setState({display: 'inline})` @CtripleU

File: submissions/cumiOyemike/react/to-do-app/src/components/Todos/Todos.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+                    />
+                    <Button
+                        disabled={!this.state.newItem}
+                        handleChange={() => this.addItem()}
+                    />
+                </header>
+                <div className="content-wrapper" style={{ display: this.state.display }}>
+                    <ul>
+                        {this.state.searchTerms.map((item, index) => {
```
Comment (ToluAkin) [inline submissions/cumiOyemike/react/to-do-app/src/components/Todos/Todos.js:90]: I think you should extract this into a function

File: submissions/cumiOyemike/react/to-do-app/src/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
```
Comment (ToluAkin) [inline submissions/cumiOyemike/react/to-do-app/src/index.js:8]: Remove Strict mode

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@CtripleU I think you should remove all tests in this application. Also, the application is not responsive,

**Comment by CtripleU** [comment]
closed

**Pull request closed without merging**